AirportTicketBooking/IsExternalInit.cs

```diff
@@ -0,0 +1,4 @@
+namespace System.Runtime.CompilerServices
+{
+    internal static class IsExternalInit {}
+}
```

Comment:
This type is required when working with Records, because records have init-only properties, and the type is missing from C#.
Otherwise the IDE will keep yelling at me xD